recipes/cpputest/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

Comment:
Since there are CMake names it's ideal if you test those here

Reply:
Thanks. Please could you clarify what I need to do?

(Like much of the code in this PR, I took this from the [corresponding file in the szip project](https://github.com/conan-io/conan-center-index/blob/b7d802fc4a3bcb447e0a53b8d37be069998c052f/recipes/szip/all/test_package/CMakeLists.txt) that's recommended as a good sample, in the [How to provide a good recipe](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#how-to-provide-a-good-recipe) section in the docs on adding new recipes...)

Reply:
Welcome to the CCI club 🤝 

I am suggestion adding the `find_package` and link to the cmake targets called out in the package info.

https://github.com/conan-io/conan-center-index/blob/5f2e16543bc83ff893de2da9a69e99676af7b5ef/recipes/backport-cpp/all/test_package/CMakeLists.txt#L7-L10

Reply:
Thank you ever so much - this was really helpful to learn about.

I think I have fixed it in e8ee75a888ce472523cef58a88e35fb281be3bbe.